jdbc-adapter/src/main/java/org/polypheny/db/adapter/jdbc/stores/PostgresqlStore.java

```diff
@@ -187,7 +190,8 @@ public Schema getCurrentSchema() {
 
     @Override
     public void addIndex( Context context, CatalogIndex catalogIndex ) {
```

Comment:
This only creates an index on one partition, not on all partitions. Drop Index needs to be adjusted as well.